README.md

```diff
@@ -591,6 +609,214 @@ ObxValue((data) => Switch(
 ),
 ```
 
+## Useful tips
+
+
+`.obs`ervables (also known as _Rx_ Types) have a wide variety of internal methods and operators.
+
+> Is very common to _believe_ that a property with `.obs` **IS** the actual value... but make no mistake!
+We avoid the Type declaration of the variable, because Dart's compiler is smart enough, and the code
+looks cleaner, but:
+```dart
+var message = 'Hello world'.obs;
+print( 'Message "$message" has Type ${message.runtimeType}');
+```
+Even if `message` _prints_ the actual String value, the Type is **RxString**!
+
+So, you can't do `message.substring( 0, 4 )`.
+You have to access the real `value` inside the _observable_: 
+The most "used way" is `.value`, but, did you know that you can also use...
+
+```dart
+final name = 'GetX'.obs;
+// only "updates" the stream, if the value is different from the current one. 
+name.value = 'Hey';
+
+// this weird (and kinda cool) assignment, updates the stream no matter what
+// it takes nulls, or same value... but rebuilds the observers.
```

Comment:
oh so assigning like that will rebuild even if the value is the same? This kinda break one of the principles of reactive state manager: only rerender when necessary.
I think this operator should respect this principle too